src/services/Subscriptions/SubscriptionQueries.ts

```diff
@@ -0,0 +1,27 @@
+import { Subscriptions, getSubscriptions } from './SubscriptionApi';
+import useErrorNotification from 'Hooks/useErrorNotification';
+import { useQuery } from 'react-query';
+
+const SUBSCRIPTION_CHECK_KEY = 'SUBSCRIPTION_CHECK_KEY';
+
+export const useFetchSubscriptionsQuery = () => {
+  const errorNotifier = useErrorNotification();
+  const { data: fetchSubscriptions, isLoading } = useQuery<Subscriptions>(
```

Comment:
Simply returning the useQuery directly should likely be done here.
In that way we can easily access any of the other variables (like isFetching, isError, error) should we wish to.

Reply:
good point, it should be done like that 👍🏼 fixed 😄